docs-chef-io/content/inspec/resources/aws_sqs_queue.md

```diff
@@ -122,20 +127,62 @@ end
 
 The control will pass if the describe returns at least one result.
 
-Use `should_not` to test the entity should not exist.
+Use `should` to test the entity should exist.
 
 ```ruby
 describe aws_sqs_queue('https://sqs.ap-southeast-2.amazonaws.com/1212121/MyQueue') do
   it { should exist }
 end
 ```
 
+Use `should_not` to test the entity should not exist.
+
 ```ruby
 describe aws_sqs_queue('https://sqs.ap-southeast-2.amazonaws.com/1212121/MyQueueWhichDoesntExist') do
   it { should_not exist }
 end
 ```
 
+### policy_statement_principal_all_permissions_enabled
+
+The control will pass if the describe returns at least one result.
+
+Use `should` to test the entity should exist.
+
+```ruby
+describe aws_sqs_queue('https://sqs.ap-southeast-2.amazonaws.com/1212121/MyQueue') do
+  it { should be_policy_statement_principal_all_permissions_enabled }
+end
+```
+
+Use `should_not` to test the entity should not exist.
+
+```ruby
+describe aws_sqs_queue('https://sqs.ap-southeast-2.amazonaws.com/1212121/MyQueueWhichDoesntExist') do
+  it { should_not be_policy_statement_principal_all_permissions_enabled }
+end
+```
+
+### policy_statement_action_all_permissions_enabled
+
+The control will pass if the describe returns at least one result.
+
+Use `should` to test the entity should exist.
+
+```ruby
+describe aws_sqs_queue('https://sqs.ap-southeast-2.amazonaws.com/1212121/MyQueue') do
+  it { should be_policy_statement_action_all_permissions_enabled }
+end
+```
+
+Use `should_not` to test the entity should not exist.
+
```

Comment:
Same with this

```suggestion
The control will pass if at least one action has all permissions enabled.

Use `should` to test that at least one action has all permissions enabled should exist.

```ruby
describe aws_sqs_queue('https://sqs.ap-southeast-2.amazonaws.com/1212121/MyQueue') do
  it { should be_policy_statement_action_all_permissions_enabled }
end
```

Use `should_not`  to test that an action with all permissions enabled should not exist.

```